wordless/theme_builder/vanilla_theme/webpack.config.js

```diff
@@ -111,17 +103,37 @@ module.exports = (env) => {
         filename: '../stylesheets/[name].css'
       }),
 
-      new ImageminPlugin({
-        test: /\.(jpe?g|png|gif|svg)$/i
+      new ImageminWebpack({
+        test: /\.(jpe?g|png|gif|svg)$/i,
+        severityError: 'warning',
+        minimizerOptions: {
+          plugins: [
+            ['gifsicle', { interlaced: true }],
+            ['jpegtran', { progressive: true }],
+            ['optipng', { optimizationLevel: 5 }],
+            [
+              'svgo',
+              {
+                plugins: [
+                  {
+                    removeViewBox: false,
+                  },
+                ],
+              },
+            ],
+          ],
+        },
       }),
 
-      new CopyWebpackPlugin([
-        {
-          from: path.join(srcDir, '/images'),
-          to: path.join(dstDir, '/images', '[path][name].[ext]'),
-          toType: 'template'
-        }
-      ])
+      new CopyWebpackPlugin({
+        patterns: [
+          {
+            from: path.join(srcDir, `/${imageFolder}`),
+            to: path.join(dstDir, `/${imageFolder}`, '[path][name].[ext]'),
```

Comment:
I think that `[path][name].[ext]` is an error 'coz you'd need a `/` betweeb `path` and `name` placeholders. Isn't it correct?

Moreover: if we'd like to follow my suggestions about `file-loader` config, we should also update here to

```suggestions
            to: path.join(dstDir, `/${imageFolder}`, '[folder]', '[name].[ext]'),
```

But this depens on the choice about the other piece of code.